src/cgame/cg_weapons.cpp

```diff
@@ -2261,6 +2324,61 @@ void CG_SelectNextInventoryItem_f()
 	}
 }
 
+/*
+===============
+CG_FindPrevWeapon
+Find previous weapon in inventory.
+===============
+*/
+weapon_t CG_FindPrevWeapon( playerState_t *ps )
+{
+	weapon_t prevWeapon = WP_NONE;
+
+	// For each weapon in the known weapon list from the end to the beginning:
+	for ( weapon_t w = WP_NUM_WEAPONS; w > WP_NONE; w = weapon_t( w - 1 ) )
```

Comment:
WP_NUM_WEAPONS is itself not a valid weapon number, so the loop should start at one less.

Reply:
Fixed